configs/kubernetes-rules-configmap.yaml

```diff
@@ -1023,3 +1023,17 @@ data:
           branch: master
         - repository: api
           branch: master
+    - destination: cluster-bootstrap
```

Comment:
cloud-provider?

Reply:
:woman_facepalming: had updated with the change locally, but forgot to push

Fixed now.